_components/form-templates.md

```diff
@@ -175,15 +168,31 @@ lead: Patterns for some of the most commonly used forms on government websites.
     <ul class="usa-content-list">
       <li>Make sure any input masking is accessible to people using screen readers. We use <a href="https://github.com/filamentgroup/politespace">Filament Group's Politespace</a> to ensure a good experience.</li>
     </ul>
+    <h4 class="usa-heading">Usability</h4>
+    <h5>When to use</h5>
+    <ul class="usa-content-list">
+      <li>When you need to be able to parse out the specific parts of a mailing address. </li>
+    </ul>
+    <h5>When to consider something else</h5>
+    <ul class="usa-content-list">
+      <li>If you need to collect addresses that may not fit this format (for example, international addresses). </li>
+      <li> If you don’t need to be able to parse out the individual pieces of an address, consider letting users type the whole thing in one large text area. </li>
+    </ul>
+    <h5>Guidance</h5>
+    <ul class="usa-content-list">
+      <li>Only label the optional inputs. Users can infer that all the others are required.</li>
+      <li>If possible, let users type their state’s abbreviation when they reach the “state” drop down menu.</li>
+      <li>Support both 5 and 9 digit ZIP codes.  Some addresses require a nine digit ZIP code. The input mask should be “#####-####” so that the text is properly formatted, regardless of whether a user enters a 5 or 9 digit ZIP code. </li>
+    </ul>
   </div>
 </div>
 
-<h2 class="usa-heading" id="login-form">Login form</h2>
+<h2 class="usa-heading" id="login-form">Sign-in form</h2>
```

Comment:
Sign-in as an adjective or a noun. Use the sign-in form to access your account. 
Sign in as a verb. Please sign in to access your account.